### ISSUE TITLE
TypeError with Webpack 5

### ISSUE THREAD

**Issue by mikejpeters** [codepunkt/webpack-license-plugin#907]
I'm running into an error while using it with Webpack `5.75.0`:

```
\ Generating browser application bundles (phase: setup)...WebpackLicensePlugin.ts:70
      compilation.hooks.optimizeChunkAssets.tapAsync(
                                            ^
TypeError: Cannot read properties of undefined (reading 'tapAsync')
    at WebpackLicensePlugin.handleCompilation (WebpackLicensePlugin.ts:70:45)<anonymous>:31:1)
```

I think this is fixed by your commits since the latest release. I tried downloading the source code and building the plugin, and then everything worked as expected.

Can you publish a new version to fix this?

Thanks for the great plugin :)

**Comment by JoaoBrlt** [comment]
I can confirm, it seems to be fixed by commit 837f2645bfe3b5772b6efe5c912ce7f0a3856c7d.
Would it be possible to get a new release @codepunkt :pray: ?

**Comment by codepunkt** [comment]
@mikejpeters @JoaoBrlt Published as `4.2.3`. Let me know if it helps!
